common/utils/util.test.js

```diff
@@ -111,3 +124,102 @@ describe('should check if date is valid', () => {
     expect(isValidDate('fish fingers', 'chips', 'peas')).toEqual(false)
   })
 })
+
+describe('should check if a status is a closed one', () => {
+  it('correctly determines a closed status', () => {
+    expect(hasClosedStatus(COMPLETED)).toEqual(true)
+    expect(hasClosedStatus(ABANDONED)).toEqual(true)
+    expect(hasClosedStatus(PARTIALLY_COMPLETED)).toEqual(true)
+    expect(hasClosedStatus(NOT_STARTED)).toEqual(false)
+    expect(hasClosedStatus(PAUSED)).toEqual(false)
+    expect(hasClosedStatus(IN_PROGRESS)).toEqual(false)
+  })
+})
+
+describe('should return the correct type for an objective', () => {
+  it('defaults to active', () => {
+    const objective = {
+      actions: [
+        {
+          status: 'IN_PROGRESS',
```

Comment:
Why not use the constant? IN_PROGRESS

Reply:
Several instances below.

Reply:
I'm not using the constants because that's not how the data comes in from the back end. It's a moot point I guess but feels weird having 'external' data being derived from our apps 'internal' data.